crates/cast/bin/cmd/wallet/mod.rs

```diff
@@ -363,18 +374,28 @@ flag to set your key via:
             Self::List(cmd) => {
                 cmd.run().await?;
             }
-            Self::DerivePrivateKey { mnemonic, mnemonic_index } => {
-                let phrase = Mnemonic::<English>::new_from_phrase(mnemonic.as_str())?.to_phrase();
-                let builder = MnemonicBuilder::<English>::default().phrase(phrase.as_str());
-                let derivation_path = "m/44'/60'/0'/0/";
-                let index = mnemonic_index.unwrap_or_default();
-                let wallet = builder
-                    .clone()
-                    .derivation_path(format!("{derivation_path}{index}"))?
-                    .build()?;
-                println!("- Account:");
-                println!("Address:     {}", wallet.address());
-                println!("Private key: 0x{}\n", hex::encode(wallet.signer().to_bytes()));
+            Self::PrivateKey { wallet, mnemonic_override, mnemonic_index_override } => {
+                let wallet = mnemonic_override
+                    .map(|mnemonic| WalletOpts {
```

Comment:
this means mnemonic_index_override is ignored if mnemonic_override is missing, so mnemonic_index_override should require mnemonic_override?

Reply:
doesn't `mnemonic_index_override` being a second arg already require `mnemonic_index` being specified?

Reply:
good point, I guess so, but not exactly sure -.-